From 07540de878a712ea0b204c4f2a8fc766eb2b6842 Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Mon, 11 Sep 2023 17:45:37 +0200
Subject: [PATCH 14/41] FROMGIT(6.7): arm64: dts: meson: u200: fix spdif output
 pin

u200 outputs spdif on GPIOAO_10, not GPIOH_4 which is used for the LCD
panel.

Fixes: cfae4eadb7cd ("arm64: dts: meson: add audio playback to u200")
Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
Link: https://lore.kernel.org/r/20230911154541.471484-2-jbrunet@baylibre.com
Signed-off-by: Neil Armstrong <neil.armstrong@linaro.org>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-u200.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-u200.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-u200.dts
index 2878e3ad7de2..8fa17a62534c 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-u200.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-u200.dts
@@ -396,7 +396,7 @@ &sd_emmc_c {
 };
 
 &spdifout {
-	pinctrl-0 = <&spdif_out_h_pins>;
+	pinctrl-0 = <&spdif_ao_out_pins>;
 	pinctrl-names = "default";
 	status = "okay";
 };
-- 
2.34.1

